### ISSUE TITLE
Only a few PDO attributes can be configured in propel.database.connections.default.attributes, in contrast to what is mentioned in docs

### ISSUE THREAD

**Issue by motin** [propelorm/Propel2#1213]
On http://propelorm.org/documentation/reference/configuration-file.html#attributes it says that all standard PDO attributes can be set as well as a Propel-specific attribute PropelPDO::PROPEL_ATTR_CACHE_PREPARES

It further mentions that attributes in the configuration file can be specified with or without the PDO:: (or PropelPDO::) constant prefix.

However, [PropelConfiguration.php restricts the the possible attributes to a few values (ATTR_EMULATE_PREPARES and ATTR_TIMEOUT)](https://github.com/propelorm/Propel2/blob/df9277744796c39c3f8dde8da57fb4304be18838/src/Propel/Common/Config/PropelConfiguration.php#L130-L131). Setting other attributes results in the following exception when for instance creating a migration (`vendor/bin/propel -vvv migration:create`):

```
  [Symfony\Component\Config\Definition\Exception\InvalidConfigurationException]
  Unrecognized options "ATTR_ERRMODE, PROPEL_ATTR_CACHE_PREPARES" under "propel.database.connections.default.attributes" 
```

Note that neither the PDO:: nor PropelPDO:: prefixes are allowed neither.

Either the docs should be updated or Propel should be configured to allow setting the mentioned attributes.

**Comment by marcj** [comment]
Yeah probably we should not limit those children. @cristianoc72 do you have an idea how to solve that elegantly?

**Comment by marcj** [comment]
Fixed via #1279.
